portal-web/test/functional/com/liferay/portalweb/macros/headless/headlessdeliveryapi/LanguagesAPI.macro

```diff
@@ -0,0 +1,43 @@
+definition {
```

Comment:
We'd better to use LanguageAPI as the macro filename.
![image](https://user-images.githubusercontent.com/1308942/188526362-6109c435-92bd-4991-95e8-c815f42e4d6a.png)